lib/client/weblogin.go

```diff
@@ -257,8 +257,8 @@ func initClient(proxyAddr string, insecure bool, pool *x509.CertPool) (*WebClien
 }
 
 // SSHAgentSSOLogin is used by tsh to fetch user credentials using OpenID Connect (OIDC) or SAML.
-func SSHAgentSSOLogin(ctx context.Context, login SSHLoginSSO) (*auth.SSHLoginResponse, error) {
-	rd, err := NewRedirector(ctx, login)
+func SSHAgentSSOLogin(ctx context.Context, login SSHLoginSSO, issueLogin IssueSSOLoginConsoleRequest) (*auth.SSHLoginResponse, error) {
```

Comment:
Like above, let's please use a config struct for this.

Reply:
Done: f0503dbbae9cbb592c3d347ae51b54c0b2f95235